applications/views.py

```diff
@@ -205,3 +205,21 @@ def export_resume(request):
         return response
     except:
         raise Http404
+
+def export_newsletter_subs(request):
+    try:
+        response = HttpResponse(open("./files/newsletter_subs.csv", 'rb').read())
```

Comment:
Why does this need to be a binary read?

Reply:
It could probably be changed to just read mode - tbh think I just used the code for exporting resumes and modified it and went from there